### DISCUSSION TITLE
Global Symfony form attribute?

### DISCUSSION THREAD

**Discussion by abunch** [symfony/symfony#47617] [Q&A]
Is there a simple way to add an attribute to all fields of a specific type across all forms in a Symfony project _without editing individual forms_?

For instance, I need to add a class to all TextType fields or a data-* attribute to all TextArea fields.

**Comment by nfragnet** [comment]
For templating concerns (adding a class, or any html attribute) I would suggest form theming instead of any change at the FormType level.

Check this part of documentation for more details : https://symfony.com/doc/current/form/form_themes.html#applying-themes-to-all-forms
